tests/update_reference_data.sh

```diff
@@ -52,22 +52,50 @@ tests[spe1_metric_vfp1]="flow vfpprod_spe1 SPE1CASE1_METRIC_VFP1 spe1_metric_vfp
 tests[base_model_1]="flow model1 BASE_MODEL_1 base_model_1"
 tests[msw_model_1]="flow model1 MSW_MODEL_1 msw_model_1"
 tests[faults_model_1]="flow model1 FAULTS_MODEL_1 faults_model_1"
+tests[base_model2]="flow model2 0_BASE_MODEL2 base_model_model2"
+tests[multregt_model2]="flow model2 1_MULTREGT_MODEL2 multregt_model2"
+tests[multxyz_model2]="flow model2 2_MULTXYZ_MODEL2 multxyz_model2"
+tests[multflt_model2]="flow model2 3_MULTFLT_MODEL2 multflt_model2"
+tests[multpvv_model2]="flow model2 4_MINPVV_MODEL2 multpvv_model2"
+tests[swatinit_model2]="flow model2 5_SWATINIT_MODEL2 swatinit_model2"
+tests[endscale_model2]="flow model2 6_ENDSCALE_MODEL2 endscale_model2"
+tests[hysteresis_model2]="flow model2 7_HYSTERESIS_MODEL2 hysteresis_model2"
+tests[multiply_tranxyz_model2]="flow model2 8_MULTIPLY_TRANXYZ_MODEL2 multiply_tranxyz_model2"
+tests[editnnc_model2]="flow model2 9_EDITNNC_MODEL2 editnnc_model2"
 tests[polymer_injectivity]="flow polymer_injectivity 2D_POLYMER_INJECTIVITY"
 tests[nnc]="flow editnnc NNC_AND_EDITNNC nnc"
 
 changed_tests=""
 for test_name in ${!tests[*]}
 do
+  
   binary=`echo ${tests[$test_name]} | awk -F ' ' '{print $1}'`
   dirname=`echo ${tests[$test_name]} | awk -F ' ' '{print $2}'`
   casename=`echo ${tests[$test_name]} | awk -F ' ' '{print $3}'`
   tname=`echo ${tests[$test_name]} | awk -F ' ' '{print $4}'`
+
+
+  echo $test_name
+  echo $binary
+  echo $dirname
+  echo $casename
+  echo $tname
+
+
   test -z "$tname" && tname=$dirname
+  
+  echo $configuration
+   
+  exit 1
+
   copyToReferenceDir \
       $configuration/build-opm-simulators/tests/results/$binary+$tname/ \
       $OPM_TESTS_ROOT/$dirname/opm-simulation-reference/$binary \
       $casename \
       EGRID INIT RFT SMSPEC UNRST UNSMRY
+  
+  
+  exit 1
```

Comment:
hmm

Reply:
Sorry about this mess. I was trying to understand how the script works and then I added some extra lines and tried to run this. Unfortunately I forgot to remote this.